docs/docs/Deployment/deployment-render.md

```diff
@@ -3,9 +3,7 @@ title: Render
 slug: /deployment-render
```

Comment:
I'm unable to comment on line 2 so I'm commenting here instead. The title "Render" does not have good SEO value. Write descriptive titles.